README.md

```diff
@@ -1 +1,44 @@
-# tdd_practice
\ No newline at end of file
+# OOP-School-Library
```

Comment:
- [ ] Please change the project's name with the name of the project on which you worked (ex: TTD practice).
- [ ] It would be better to add a description to your readme, this helps document your project professionally.